docs/06-concepts/18-testing/01-get-started.md

```diff
@@ -0,0 +1,138 @@
+# Get started
+
+Serverpod provides simple but feature rich test tools to make testing your backend a breeze.
+
+<details>
+<summary> Have an existing project? Follow these steps first!</summary>
+<p>
+For existing projects, a few extra things need to be done:
+1. Add the `server_test_tools_path` key to `config/generator.yaml`. Without this key, the test tools file is not generated. The default location for the generated file is `integration_test/test_tools/serverpod_test_tools.dart`, but this can be set to any path (though should be outside of `lib` as per Dart's test conventions).
+
+2. New projects now come with a test profile in `docker-compose.yaml`. This is not strictly mandatory, but is recommended to ensure that the testing state is never polluted. Add the snippet below to the `docker-compose.yaml` file in the server directory:
```

Comment:
Suggestion: Also link to the complete file after the modifications.

Reply:
Do you mean linking to the file in the Serverpod repo?

Reply:
Then I'll have to wait for the templates PR to be merged first.

Reply:
Yes, or possibly have it as a foldout in the foldout? Just so that it is possible to do a complete copy paste of the file if required.

Reply:
Makes sense!